integrations/runllm-widget/gitbook-manifest.yaml

```diff
@@ -46,6 +46,11 @@ contentSecurityPolicy:
 configurations:
     site:
         properties:
+            native_ai_experience:
+                type: boolean
+                title: Native AI experience
+                description: Set to "true" to enable the built-in AI experience. Default is "false".
```

Comment:
Should we mention that this will hide the RunLLM button? For example:

"Set to "true" to enable triggering RunLLM from githbook's built-in AI experience. This will hide the RunLLM trigger button. Default is "false"."